dCommon/dConfig.cpp

```diff
@@ -10,16 +10,31 @@ dConfig::dConfig(const std::string& filepath) {
 	LoadConfig();
 }
 
+std::filesystem::path getConfigDir() {
+	std::filesystem::path config_dir = BinaryPathFinder::GetBinaryDir();
+	if (const char* env_p = std::getenv("DLU_CONFIG_DIR")) {
```

Comment:
nit: what would you think about this being a string_view instead of a raw const char*?

Reply:
It's the actual return of that fn and it works with the nullptr check, its a std function that we can't change, and we immediately pass it to /=